app/src/test/java/com/keylesspalace/tusky/util/AbsoluteTimeFormatterTest.kt

```diff
@@ -1,12 +1,35 @@
 package com.keylesspalace.tusky.util
 
+import org.junit.AfterClass
 import org.junit.Assert.assertEquals
+import org.junit.BeforeClass
 import org.junit.Test
 import java.time.Instant
-import java.util.Date
-import java.util.TimeZone
+import java.util.*
```

Comment:
I’m not sure of this import with `*` but this seems to pass `klintFormat` so I let it.